sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -1,16 +1,539 @@
+)
+let client = AzureCommunicationChatClient(baseUrl: baseUrl, authPolicy: authPolicy, withOptions: options)
+```
+
+# Key concepts
+
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages.
+
+Once you initialize an `AzureCommunicationChatClient` class, you can perform the following chat operations:
+
+## Thread Operations
+
+- [Create a thread](#create-a-thread)
+- [Get a thread](#get-a-thread)
+- [List threads](#list-threads)
+- [Update a thread](#update-a-thread)
+- [Delete a thread](#delete-a-thread)
+
+## Message Operations
+
+- [Send a message](#send-a-message)
+- [Get a message](#get-a-message)
+- [List messages](#list-messages)
+- [Update a message](#update-a-message)
+- [Delete a message](#delete-a-message)
+
+## Thread Member Operations
+
+- [Get thread members](#get-thread-members)
+- [Add thread members](#add-thread-members)
+- [Remove a thread member](#remove-a-thread-member)
+
+## Events Operations
+
+- [Send a typing notification](#send-a-typing-notification)
+- [Send read receipt](#send-read-receipt)
+- [Get read receipts](#get-read-receipts)
+
+# Examples
+
+## Thread Operations
+
+### Create a thread
+
+Use the `create` method to create a thread.
+
+- `CreateChatThreadRequest` is the model to pass to this method.
+- `topic` is used to provide a topic for the thread.
+- `members` is used to list the `ChatThreadMember` to be added to the thread.
+- `id`, required, is the `CommunicationUser.identifier` you created before. Refer to [User and User Access Tokens](#User-and-User-Access-Tokens).
+- `displayName`, optional, is the display name for the thread member.
+- `shareHistoryTime`, optional, is the time from which the chat history is shared with the member.
+
+`MultiStatusResponse` is the result returned from creating a thread. It has a 'multipleStatus' property which represents a list of `IndividualStatusResponse`.
+
+```swift
+let thread = CreateChatThreadRequest(
+    topic: "General",
+    members: [ChatThreadMember(
+        id: <userId>,
+        // Id of this needs to match with the Auth token
+        displayName: "initial member"
+    )]
+)
+
+client.create(chatThread: thread) { result, _ in
+    switch result {
+    case let .success(createThreadResponse):
+        var threadId: String? = nil
+        for response in createThreadResponse.multipleStatus ?? [] {
+            assert(response.id != nil, "Thread id should not be nil")
+
+            if response.id?.hasSuffix("@thread.v2") ?? false,
+                response.type ?? "" == "Thread" {
+                threadId = response.id
+            }
+        }
+        guard let thread = threadId else { fatalError("ThreadId not found.") }
+        print("<- SUCCESS! in createThread. Thread id= \(thread)")
```

Comment:
I'd get rid of these print statements as well and just put a TODO or something to say "TODO: Take further action"